### DISCUSSION TITLE
Error al correr "make clean && make"

### DISCUSSION THREAD

**Discussion by IsabellaKPM** [IIC2133-PUC/2024-2#31] [Tarea 0] [answered]
Hola! Estaba tratando de recompilar mi código para probarlo, pero al hacer `make clean && make` o sólo `make` me lanza el siguiente error:
![image](https://github.com/user-attachments/assets/35097fbc-f390-441e-9daf-81b66460b3f1)
No sé qué podría estar pasando. Cuando compilé por primera vez salió todo ok.
Adjunto también una imagen de los archivos de la carpeta:
![image](https://github.com/user-attachments/assets/5f16df6e-d47f-4fa5-8c23-9cca50380610)
Creo que el problema es que no puede eliminar el archivo `main.o` que está en `obj/puffle` y de hecho yo tampoco lo puedo eliminar manual por vscode porque me dice que no tengo los permisos y cuando lo elimino desde el explorador de archivos sólo vuelve a aparecer.
Añado igualmente que estoy trabajando con Ubuntu (porque no pude configurarlo de otra forma).

**Comment by tomastrivino** [comment, marked as answer]
Hola. Qué extraño. Prueba ejecutando `chmod 777 -R obj`  o `sudo chmod 777 -R obj` a ver si se arregla.

**Reply by tomastrivino** [reply]
Ese comando debería dar los permisos a todos los usuarios del sistema para manipular dicha carpeta.

**Reply by IsabellaKPM** [reply]
Síi, con `sudo chmod 777 -R obj` se arregló. Muchas graciaas

**Comment by Mikson16** [comment]
A  mi también me ocurre un problema al correr clean && make, adjunto una imagen del error que me aparece 
![image](https://github.com/user-attachments/assets/1fd3a4b5-7d92-4297-9dec-48f9c6784ee8)

**Reply by tomastrivino** [reply]
Hola,
@Mikson16 es `make clean && make`.
Saludos!

**Reply by Mikson16** [reply]
Oooh, muchas gracias, no sabía :3

**Reply by tomastrivino** [reply]
No worries :3
